### PR TITLE
Feature/student product support

### PR DESCRIPTION
**Pull request by palmfjord**
Credz to @hedvigoscar for doing the heavy lifting 🏋️‍♀️

### CODE REVIEW

File: src/pages/Chat/steps/CreateOffer.tsx [hunk 1]
```diff
@@ -6,6 +6,11 @@ import { SessionContainer } from 'containers/SessionContainer'
 import * as React from 'react'
 import styled from 'react-emotion'
 import { Update } from 'react-lifecycle-components'
+import { qualifiesForStudentInsurance } from 'utils/insuranceDomainUtils'
+import {
+  InputValidationError,
+  UserCheckbox,
+} from '../../../components/userInput/UserResponse'
```
Comment (palmfjord) [inline src/pages/Chat/steps/CreateOffer.tsx]: ../.../../././././../././. :D

Comment (frapperino) [reply src/pages/Chat/steps/CreateOffer.tsx]: 😭

File: src/components/userInput/UserResponse.tsx [hunk 2]
```diff
@@ -46,41 +46,111 @@ export const UserTextInput = styled('input')(
...
+  transition: 'opacity 150ms',
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  width: '.6em',
+  transform: 'translateX(-50%) translateY(-50%)',
+  fill: colors.WHITE,
 
-const UserSelectInputWrapper = styled(UserTextInput)((_: WithMaxWidth) => ({
+  'input:checked + &': {
```
Comment (Iamsamwhoami) [inline src/components/userInput/UserResponse.tsx]: Would be nicer to use react than CSS magic to do this, change it if you want :)

File: src/pages/Offer/sections/InsuredAmount.tsx [hunk 3]
```diff
@@ -52,37 +53,45 @@ const InfoText = styled('div')({
...
+}) => (
   <Card>
     <HeaderWrapper>
       <TranslationsConsumer textKey="OFFER_INSURED_AMOUNT_TITLE">
         {(title) => <Header>{title}</Header>}
       </TranslationsConsumer>
     </HeaderWrapper>
     <Table>
-      {rows.map((row, index) => (
+      {rows(isStudentInsurance(insuranceType)).map((row, index) => (
```
Comment (hedvigoscar) [inline src/pages/Offer/sections/InsuredAmount.tsx]: 📸

File: src/components/userInput/UserResponse.tsx (hunk 2 above)
Comment (palmfjord) [reply src/components/userInput/UserResponse.tsx]: Why? :) it's a pure style thing so imo it should be css

File: src/pages/Offer/sections/InsuredAmount.tsx (hunk 3 above)
Comment (palmfjord) [reply src/pages/Offer/sections/InsuredAmount.tsx]: ?

Comment (hedvigoscar) [reply src/pages/Offer/sections/InsuredAmount.tsx]: I knew this was a little too unclear. I wanted to convey that it was clean enough to be picture-worthy 🤷‍♂️

Comment (palmfjord) [reply src/pages/Offer/sections/InsuredAmount.tsx]: haha right 😄

**Pull request merged**